### PR TITLE
Bump v1.3.0

### PR DESCRIPTION
**Pull request by lucklove**
### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

Side effects

 - Possible performance regression
 - Increased code complexity
 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

Release notes:

```release-note
NONE
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,43 @@
...
+### New Features
+
+- Modify TiFlash's query memory limit from 10GB to 0(unlimited) in tiup-playground ([#907](https://github.com/pingcap/tiup/pull/907), [@LittleFall](https://github.com/LittleFall))
+- Import configuration into topology meta when migrating a cluster from Ansible ([#766](https://github.com/pingcap/tiup/pull/766), [@yuzhibotao](https://github.com/yuzhibotao))
+  - Before, we stored imported ansible config in ansible-imported-configs which is hidden for users, in this release, we merge the configs into meta.yaml so that the user can see the config with the command `tiup cluster edit`
+- Enhance the `tiup mirror` command ([#860](https://github.com/pingcap/tiup/pull/860), [@lucklove](https://github.com/lucklove))
+  - **Support merge two or more mirrors into one**
+  - Support publish component to local mirror besides remote mirror
+  - Support add component owner to local mirror
+- Support deploy cluster with hostname besides ip address (EXPERIMENTAL) ([#948](https://github.com/pingcap/tiup/pull/948),[#949](https://github.com/pingcap/tiup/pull/949), [@fln](https://github.com/fln))
```
Comment (AstroProfundis) [inline CHANGELOG.md:14]:
```suggestion
- Partial support deploy cluster with hostname besides ip address (**EXPERIMENTAL**) ([#948](https://github.com/pingcap/tiup/pull/948),[#949](https://github.com/pingcap/tiup/pull/949), [@fln](https://github.com/fln))
  - Not usable for production, as there would be issue if a hostname resolves to a new IP address after deployment
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,43 @@
...
+- Support setting custom timeout for waiting instances up in tiup-playground ([#968](https://github.com/pingcap/tiup/pull/968), [@unbyte](https://github.com/unbyte))
+- Support check and disable THP in tiup cluster check ([#964](https://github.com/pingcap/tiup/pull/964), [@anywhy](https://github.com/anywhy))
```
Comment (AstroProfundis) [inline CHANGELOG.md:16]:
```suggestion
- Support check and disable THP in `tiup cluster check` ([#964](https://github.com/pingcap/tiup/pull/964), [@anywhy](https://github.com/anywhy))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,43 @@
...
+- Support sign remote manifest and rotate root.json ([#967](https://github.com/pingcap/tiup/pull/967), [@lucklove](https://github.com/lucklove))
+
+### Fixes
+
+- Fix the issue that public key created by TiUP not removed after the cluster destroyed ([#910](https://github.com/pingcap/tiup/pull/910), [@9547](https://github.com/9547))
+- Fix the issue that user defined username and password not imported from tidb-ansible cluster correctly ([#937](https://github.com/pingcap/tiup/pull/937), [@AstroProfundis](https://github.com/AstroProfundis))
+- Fix the issue that tiup-playground not quit components with correct order: TiDB -> TiKV -> PD ([#933](https://github.com/pingcap/tiup/pull/933), [@unbyte](https://github.com/unbyte))
```
Comment (AstroProfundis) [inline CHANGELOG.md:23]:
```suggestion
- Fix the issue that tiup-playground not quiting components with correct order: TiDB -> TiKV -> PD ([#933](https://github.com/pingcap/tiup/pull/933), [@unbyte](https://github.com/unbyte))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,43 @@
...
+- Fix the issue that tikv reports wrong advertise address when `--status-addr` is set to a wildcard address like `0.0.0.0` ([#951](https://github.com/pingcap/tiup/pull/951), [@lucklove](https://github.com/lucklove))
+- Make error message of monitor port conflict more readable ([#966](https://github.com/pingcap/tiup/pull/966), [@JaySon-Huang](https://github.com/JaySon-Huang))
```
Comment (AstroProfundis) [inline CHANGELOG.md:25]: This should be an improvement

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,43 @@
...
+
+### Improvements
+
+- Enable memory buddyinfo monitoring on node_exporter to collect exposes statistics of memory fragments ([#904](https://github.com/pingcap/tiup/pull/904), [@9547](https://github.com/9547))
+- Move error logs dumped by tiup-dm and tiup-cluster to `${TIUP_HOME}/logs` ([#908](https://github.com/pingcap/tiup/pull/908), [@9547](https://github.com/9547))
+- Allow run pure TiKV (without TiDB) cluster in tiup-playground ([#926](https://github.com/pingcap/tiup/pull/926), [@sticnarf](https://github.com/sticnarf))
+- Add confirm stage for upgrade action ([#963](https://github.com/pingcap/tiup/pull/963), [@Win-Man](https://github.com/Win-Man))
+- Omit debug log from console output in tiup-cluster ([#977](https://github.com/pingcap/tiup/pull/977), [@AstroProfundis](https://github.com/AstroProfundis))
+- Prompt list of paths to be delete before processing in the clean action of tiup-cluster ([#981](https://github.com/pingcap/tiup/pull/981), [#993](https://github.com/pingcap/tiup/pull/993), [@AstroProfundis](https://github.com/AstroProfundis))
```
Comment (9547) [inline CHANGELOG.md:39]: s/delete/deleted/g

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,43 @@
...
+- Fix the issue that tikv reports wrong advertise address when `--status-addr` is set to a wildcard address like `0.0.0.0` ([#951](https://github.com/pingcap/tiup/pull/951), [@lucklove](https://github.com/lucklove))
+- Make error message of monitor port conflict more readable ([#966](https://github.com/pingcap/tiup/pull/966), [@JaySon-Huang](https://github.com/JaySon-Huang))
+- Fix the issue that the cluster can't start when there are multiple prometheus ([#972](https://github.com/pingcap/tiup/pull/972), [@9547](https://github.com/9547))
+- Fix the issue that prometheus doesn't reload target after scale-in action ([#958](https://github.com/pingcap/tiup/pull/958), [@9547](https://github.com/9547))
+- Fix the issue that the config file for tiflash missing in tiup-palyground ([#969](https://github.com/pingcap/tiup/pull/969), [@unbyte](https://github.com/unbyte))
+- Fix the issue that the tiflash doesn't create err log while numactl is missing ([#984](https://github.com/pingcap/tiup/pull/984), [@lucklove](https://github.com/lucklove))
```
Comment (9547) [inline CHANGELOG.md:29]: s/tiflash/TiFlash/g 
s/while/when/g

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,43 @@
...
+- Fix the issue that tiup-playground not quit components with correct order: TiDB -> TiKV -> PD ([#933](https://github.com/pingcap/tiup/pull/933), [@unbyte](https://github.com/unbyte))
+- Fix the issue that tikv reports wrong advertise address when `--status-addr` is set to a wildcard address like `0.0.0.0` ([#951](https://github.com/pingcap/tiup/pull/951), [@lucklove](https://github.com/lucklove))
+- Make error message of monitor port conflict more readable ([#966](https://github.com/pingcap/tiup/pull/966), [@JaySon-Huang](https://github.com/JaySon-Huang))
+- Fix the issue that the cluster can't start when there are multiple prometheus ([#972](https://github.com/pingcap/tiup/pull/972), [@9547](https://github.com/9547))
+- Fix the issue that prometheus doesn't reload target after scale-in action ([#958](https://github.com/pingcap/tiup/pull/958), [@9547](https://github.com/9547))
+- Fix the issue that the config file for tiflash missing in tiup-palyground ([#969](https://github.com/pingcap/tiup/pull/969), [@unbyte](https://github.com/unbyte))
```
Comment (9547) [inline CHANGELOG.md:28]: s/tiflash/TiFlash/g

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,43 @@
...
+- Fix the issue that user defined username and password not imported from tidb-ansible cluster correctly ([#937](https://github.com/pingcap/tiup/pull/937), [@AstroProfundis](https://github.com/AstroProfundis))
+- Fix the issue that tiup-playground not quit components with correct order: TiDB -> TiKV -> PD ([#933](https://github.com/pingcap/tiup/pull/933), [@unbyte](https://github.com/unbyte))
+- Fix the issue that tikv reports wrong advertise address when `--status-addr` is set to a wildcard address like `0.0.0.0` ([#951](https://github.com/pingcap/tiup/pull/951), [@lucklove](https://github.com/lucklove))
+- Make error message of monitor port conflict more readable ([#966](https://github.com/pingcap/tiup/pull/966), [@JaySon-Huang](https://github.com/JaySon-Huang))
+- Fix the issue that the cluster can't start when there are multiple prometheus ([#972](https://github.com/pingcap/tiup/pull/972), [@9547](https://github.com/9547))
+- Fix the issue that prometheus doesn't reload target after scale-in action ([#958](https://github.com/pingcap/tiup/pull/958), [@9547](https://github.com/9547))
```
Comment (9547) [inline CHANGELOG.md:27]: s/prometheus/Prometheus/g

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,43 @@
...
+
+### Fixes
+
+- Fix the issue that public key created by TiUP not removed after the cluster destroyed ([#910](https://github.com/pingcap/tiup/pull/910), [@9547](https://github.com/9547))
+- Fix the issue that user defined username and password not imported from tidb-ansible cluster correctly ([#937](https://github.com/pingcap/tiup/pull/937), [@AstroProfundis](https://github.com/AstroProfundis))
+- Fix the issue that tiup-playground not quit components with correct order: TiDB -> TiKV -> PD ([#933](https://github.com/pingcap/tiup/pull/933), [@unbyte](https://github.com/unbyte))
+- Fix the issue that tikv reports wrong advertise address when `--status-addr` is set to a wildcard address like `0.0.0.0` ([#951](https://github.com/pingcap/tiup/pull/951), [@lucklove](https://github.com/lucklove))
+- Make error message of monitor port conflict more readable ([#966](https://github.com/pingcap/tiup/pull/966), [@JaySon-Huang](https://github.com/JaySon-Huang))
+- Fix the issue that the cluster can't start when there are multiple prometheus ([#972](https://github.com/pingcap/tiup/pull/972), [@9547](https://github.com/9547))
```
Comment (9547) [inline CHANGELOG.md:26]: s/prometheus/Prometheus/g

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,5 +1,43 @@
...
+- Support check and disable THP in tiup cluster check ([#964](https://github.com/pingcap/tiup/pull/964), [@anywhy](https://github.com/anywhy))
+- Support sign remote manifest and rotate root.json ([#967](https://github.com/pingcap/tiup/pull/967), [@lucklove](https://github.com/lucklove))
+
+### Fixes
+
+- Fix the issue that public key created by TiUP not removed after the cluster destroyed ([#910](https://github.com/pingcap/tiup/pull/910), [@9547](https://github.com/9547))
+- Fix the issue that user defined username and password not imported from tidb-ansible cluster correctly ([#937](https://github.com/pingcap/tiup/pull/937), [@AstroProfundis](https://github.com/AstroProfundis))
+- Fix the issue that tiup-playground not quit components with correct order: TiDB -> TiKV -> PD ([#933](https://github.com/pingcap/tiup/pull/933), [@unbyte](https://github.com/unbyte))
+- Fix the issue that tikv reports wrong advertise address when `--status-addr` is set to a wildcard address like `0.0.0.0` ([#951](https://github.com/pingcap/tiup/pull/951), [@lucklove](https://github.com/lucklove))
```
Comment (9547) [inline CHANGELOG.md:25]: s/tikv/TiKV/g

File: CHANGELOG.md (hunk 3 above)
Comment (9547) [inline CHANGELOG.md]: that could not exit components in the correct order

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,5 +1,43 @@
...
+  - Support publish component to local mirror besides remote mirror
+  - Support add component owner to local mirror
+- Support deploy cluster with hostname besides ip address (EXPERIMENTAL) ([#948](https://github.com/pingcap/tiup/pull/948),[#949](https://github.com/pingcap/tiup/pull/949), [@fln](https://github.com/fln))
+- Support setting custom timeout for waiting instances up in tiup-playground ([#968](https://github.com/pingcap/tiup/pull/968), [@unbyte](https://github.com/unbyte))
+- Support check and disable THP in tiup cluster check ([#964](https://github.com/pingcap/tiup/pull/964), [@anywhy](https://github.com/anywhy))
+- Support sign remote manifest and rotate root.json ([#967](https://github.com/pingcap/tiup/pull/967), [@lucklove](https://github.com/lucklove))
+
+### Fixes
+
+- Fix the issue that public key created by TiUP not removed after the cluster destroyed ([#910](https://github.com/pingcap/tiup/pull/910), [@9547](https://github.com/9547))
```
Comment (9547) [inline CHANGELOG.md:22]: Fixed the issue that the public key created by TiUP was not removed after the cluster was destroyed

**Review by 9547 — APPROVED** [review]
/lgtm

File: CHANGELOG.md [hunk 12]
```diff
@@ -1,5 +1,43 @@
...
+- Fix the issue that user defined grafana username and password not imported from tidb-ansible cluster correctly ([#937](https://github.com/pingcap/tiup/pull/937), [@AstroProfundis](https://github.com/AstroProfundis))
+- Fix the issue that playground cluster not quiting components with correct order: TiDB -> TiKV -> PD ([#933](https://github.com/pingcap/tiup/pull/933), [@unbyte](https://github.com/unbyte))
+- Fix the issue that TiKV reports wrong advertise address when `--status-addr` is set to a wildcard address like `0.0.0.0` ([#951](https://github.com/pingcap/tiup/pull/951), [@lucklove](https://github.com/lucklove))
+- Fix the issue that Prometheus doesn't reload target after scale-in action ([#958](https://github.com/pingcap/tiup/pull/958), [@9547](https://github.com/9547))
+- Fix the issue that the config file for TiFlash missing in playground cluster ([#969](https://github.com/pingcap/tiup/pull/969), [@unbyte](https://github.com/unbyte))
+- Fix the issue that the TiFlash doesn't create err log when numactl is missing ([#984](https://github.com/pingcap/tiup/pull/984), [@lucklove](https://github.com/lucklove))
```
Comment (King-Dylan) [inline CHANGELOG.md:28]: - Fix Tilfash startup failure without stderr output When numa is enabled but numactl cannot be found ([#984](https://github.com/pingcap/tiup/pull/984), [@lucklove](https://github.com/lucklove))

File: CHANGELOG.md (hunk 6 above)
Comment (King-Dylan) [inline CHANGELOG.md]: - Fix Tilfash startup failure without stderr output When numa is enabled but numactl cannot be found ([#984](https://github.com/pingcap/tiup/pull/984), [@lucklove](https://github.com/lucklove))

File: CHANGELOG.md [hunk 13]
```diff
@@ -1,5 +1,43 @@
...
+- Fix the issue that tikv reports wrong advertise address when `--status-addr` is set to a wildcard address like `0.0.0.0` ([#951](https://github.com/pingcap/tiup/pull/951), [@lucklove](https://github.com/lucklove))
+- Make error message of monitor port conflict more readable ([#966](https://github.com/pingcap/tiup/pull/966), [@JaySon-Huang](https://github.com/JaySon-Huang))
+- Fix the issue that the cluster can't start when there are multiple prometheus ([#972](https://github.com/pingcap/tiup/pull/972), [@9547](https://github.com/9547))
+- Fix the issue that prometheus doesn't reload target after scale-in action ([#958](https://github.com/pingcap/tiup/pull/958), [@9547](https://github.com/9547))
+- Fix the issue that the config file for tiflash missing in tiup-palyground ([#969](https://github.com/pingcap/tiup/pull/969), [@unbyte](https://github.com/unbyte))
+- Fix the issue that the tiflash doesn't create err log while numactl is missing ([#984](https://github.com/pingcap/tiup/pull/984), [@lucklove](https://github.com/lucklove))
+- Fix the issue that deploy failed when remote is using zsh ([#982](https://github.com/pingcap/tiup/pull/982), [@9547](https://github.com/9547))
```
Comment (King-Dylan) [inline CHANGELOG.md]: - Fix the issue that the deployment environment fails to copy config file when zsh is configured ([#982](https://github.com/pingcap/tiup/pull/982), [@9547](https://github.com/9547))

**Pull request merged**